app/src/main/java/de/dhbw/corona_world_app/Cache.java

```diff
@@ -2,51 +2,33 @@
+        for (String countryName:countryNames) {
+            countries.put(countryName,new Country());
+        }
     }
 
-    public static void deleteCountry(Country country){
-        countries.remove(country);
+    public static Set<String> getCountryNames() {
+        return countries.keySet();
     }
```

Comment:
Bitte Namen der Länder in API Management ablegen. Nur Länder in Cache legen, wo die Daten zu dem Land im Cache sind.